docs/install/serving/install-serving-with-yaml.md

```diff
@@ -31,27 +31,28 @@ Follow the procedure for the networking layer of your choice:
 <!-- TODO: Link to document/diagram describing what is a networking layer.  -->
 <!-- This indentation is important for things to render properly. -->
 
-=== "Kourier (Choose this if you are not sure)"
+=== "Istio (default)"
```

Comment:
Is Istio the default layer when installing with YAML Istio? I don't know if the default networking layer here is the same as when installing with the Operator.

Reply:
Samia is correct. Istio is not default when installing with YAML.